demo/kserve/deploy-remove.md

```diff
@@ -42,14 +43,16 @@ Note: The **flan-t5-small** LLM model has been containerized into an S3 MinIO bu
    a. Create a new namespace.
 
    ```bash
-   export TEST_NS=kserve-demo
+   export TEST_NS=kserve-demo"-${INF_PROTO}"
```

Comment:
Not sure we want to restrict this specifically to its own NS

Also, nit, can be `TEST_NS="kserve-demo-${INF_PROTO}"`

Reply:
I agree with @Xaenalt. The gRPC and rest should be supported in the same namespace. If it is not working well, we have to fix it

Reply:
I moved code to a single kserve-demo namespace.
note that when using the step-by-step documentation,  we may create servingruntimes and isvc for both HTTP and gRPC (at the same time) and it is working fine.